src/components/Banners/MainBanner/index.js

```diff
@@ -1,11 +1,14 @@
 import React from "react";
-import Icon from "../../Icon/Icon";
+
+//components
 import SearchBanner from "../SearchBanner/SearchBanner";
```

Comment:
These imports are reported as unresolved by linter warning, consider fixing them - otherwise it might crash the app.

Reply:
import file paths have been corrected and resolved.